apps/search/__init__.py

```diff
@@ -149,15 +148,7 @@ def _get_types(self, req):
         """Get document types for the given query."""
         args = getattr(req, 'args', {})
         repos = args.get('repo')
-
-        # If not repos were supplied, return the default repos
-        if repos is None:
-            return self.repos.copy()
-        else:
-            repos = repos.split(',')
-
-            # If the repos array is still empty after filtering, then return the default repos
-            return [repo for repo in repos if repo in self.repos] or self.repos.copy()
+        return es_utils.get_doc_types(repos)
```

Comment:
Can we still use `self.repos` here please and pass that as the second parameter to `es_utils.get_doc_types` as a default value?

Reason being is that Analytics inherits from this class and makes use of self.repos (defaulting to the one that this Search class implements). Modifying `get_doc_types` for a default value will also make it usable elsewhere (if required).